.github/scripts/iam_risc0_cache.sh

```diff
@@ -0,0 +1,2 @@
+cp ../../../../_aws/certificate.pem $HOME/.local/aws/certificate.pem
+cp ../../../../_aws/runner_key.pem $HOME/.local/aws/runner_key.pem
```

Comment:
Can we move this to ansible/host-config rather than being in a github action?

Reply:
Absolutely. The files are on the runners already, so this script is actually no longer necessary and was removed.

Instead, the files are now just referenced in relative a path when used to set credentials:
https://github.com/risc0/risc0/blob/morgan/external-prs/.github/actions/sccache/action.yml#L127-129

```
env:
  ...
  IAM_RA_PATH: ../../_aws
run: |
  aws configure set credential_process "$HOME/.local/aws/aws_signing_helper credential-process --trust-anchor-arn $TRUST_ANCHOR_ARN --profile-arn $PROFILE_ARN --role-arn $ROLE_ARN --certificate $IAM_RA_PATH/certificate.pem --private-key $IAM_RA_PATH/runner_key.pem" --profile default
```

Reply:
https://github.com/risc0/devops/pull/225